test/downloadGTest.cmake.in

```diff
@@ -13,7 +13,7 @@ project(googletest-download NONE)
 include(ExternalProject)
 ExternalProject_Add(googletest
     GIT_REPOSITORY    https://github.com/google/googletest.git
-    GIT_TAG           release-1.8.0
+    GIT_TAG           release-1.10.0
```

Comment:
Furthermore a warning was treated as error, which broke my Visual Studio build. Updating gtest resolved the issue.

Reply:
:+1: